src/framework/components/anim/component-binder.js

```diff
@@ -87,16 +100,37 @@ class AnimComponentBinder extends DefaultAnimBinder {
                 if (!propertyComponent) {
                     var entityPath = AnimBinder.splitPath(propertyLocation.entityPath[0], '/');
                     propertyComponent = this.nodes[entityPath[entityPath.length - 1] || ""];
+
+                    // #ifdef DEBUG
+                    var fallbackGraphPath = (entityPath[entityPath.length - 1] || "") + '/' + propertyLocation.propertyPath;
+                    if (this._visitedFallbackGraphPaths[fallbackGraphPath]) {
+                        console.warn('Multiple nodes with the path ' + fallbackGraphPath + ' are present in the entities graph which may result in the incorrect binding of animations');
```

Comment:
if you can easily print the name of the entity the anim component is on, that would be useful as well

Reply:
Sure i'll add that in